src/en/juju-ha.md

```diff
@@ -51,16 +51,10 @@ state servers nor remove the failed ones. However, as long as more than half of
 the original number of state servers remain available you can manually recover.
 The process is detailed below.
 
-1. Run `juju ensure-availability`.
-1. Verify that the output of `juju status` shows a value of `has-vote` for the
```

Comment:
Urgh. I think that I can't remove because I only have three and a majority must remain to do this recovery. Testing again with more controller members.